### DISCUSSION TITLE
[AVRAM] I saw no different when specify auto_inner_join option when chainable with left_join_### ?

### DISCUSSION THREAD

**Discussion by zw963** [luckyframework/lucky#1800] [Q&A]
Check following example:

```crystal
CompanyQuery.new.left_join_manufactories.where_manufactories(ManufactoryQuery.new.name("工厂"), auto_iauto_inner_joinnner_join: true)
```

```crystal
CompanyQuery.new.left_join_manufactories.where_manufactories(ManufactoryQuery.new.name("工厂"), auto_iauto_inner_joinnner_join: false)
```

Both of them will generated(expected) SQL like this: (both of them use LEFT JOIN)

```
SELECT COLUMNS FROM companies LEFT JOIN manufactories ON companies.id = manufactories.company_id WHERE manufactories.name = $1", args: ["工厂"]
```

so, if this is true, why we mention this in document for must specify `auto_iauto_inner_joinnner_join: false` when use left outer join [here](https://luckyframework.org/guides/database/querying-records#joins)?

Thanks

--------------

It menthion [here](https://luckyframework.org/guides/database/querying-records#using-with-associations) too

**Comment by jwoertink** [comment]
Hmm :thinking:  ... Maybe `auto_inner_join` is not needed anymore? I haven't used it in a very long time, so it may just be leftover from old code.... I'm not sure.

**Reply by zw963** [reply]
So, we need fix document, right?

**Reply by jwoertink** [reply]
I'm not too familiar with that part of the code. I'll try and look it over when I have a moment, but I don't want to change documentation until we know for sure.
